src/_tutorials/server/cmdline.md

```diff
@@ -572,7 +572,9 @@ For more classes, functions, and properties,
 consult to the API reference for
 [dart:io,]({{ioAPI}}/dart-io-library.html)
 [dart:convert,]({{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-convert/dart-convert-library.html)
-and the [args]({{argsAPI}}/args-library.html) package.
+and the [args]({{argsAPI}}/args-library.html) package.  
+
+For an example of a command line app, see the [command_line](https://github.com/dart-lang/samples/tree/master/command_line) sample.
```

Comment:
One small suggestion is to modify the addition to use [semantic line breaks](https://github.com/dart-lang/site-shared/blob/master/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks). We use them to make reading and editing in source more clear. I also split the link declaration in the suggestion to make the line more readable.

```suggestion
For another example of a command line app, 
see the [command_line][] sample.

[command_line]: https://github.com/dart-lang/samples/tree/master/command_line
```

Reply:
Thanks for making the updates!

Reply:
Thanks for the feedback @parlough! I made these changes. Let me know if there's anything further to do.